python/cuml/decomposition/pca.pyx

```diff
@@ -451,7 +451,7 @@ class PCA(Base):
 
         return X_new
 
-    def inverse_transform(self, X):
+    def inverse_transform(self, X, convert_dtype=False):
```

Comment:
I don't think we need convert_dtype parameter here because the only supported format is float or double, and the data has to be converted if the input has a different type than these two.

Reply:
I believe this to be a good approach to make it more painless for users. It's likely rarely ever going to be invoked and if I'm understanding this properly, it's turned off by default. 

There's nothing more frustrating than being in mid-exploration and having your tools start acting up because you didn't realize the types were wrong in the model you built. This provides an option for users to say "You know what, I don't even care about types at this point, I just need to keep exploring while the ideas are fresh in my head". The struggle is real.